src/components/AnnotationPopover/Overlay.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+
+import OverlayComponent from 'box-react-ui/lib/components/flyout/Overlay';
+
+type Props = {
+    shouldDefaultFocus: boolean,
+    children: React.Node
+};
+
+const Overlay = ({ shouldDefaultFocus, children }: Props) => {
+    if (shouldDefaultFocus) {
```

Comment:
so if default focus, then use the box-react-ui overlay component, otherwise just render the comments in a div? Is this the issue of focus of the overlay messing up rangy?

Reply:
Yup